source/threads.tex

```diff
@@ -11910,6 +11895,21 @@
   explicit packaged_task(F&& f);
 \end{itemdecl}
 
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+packaged_task(allocator_arg, allocator<int>(), std::forward<F>(f)).
```

Comment:
Stray period in code block.

Maybe use `\tcode` instead of a code block if there should be a sentence period here?

Reply:
Fixed